nativelib/src/main/resources/platform.c

```diff
@@ -0,0 +1,20 @@
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+int scalanative_platform_is_windows() {
```

Comment:
I'd suggest to make this more generic as a first step by using the OS specific #defines to provide a value for the os.name property. Then in Scala this can be mapped to isWindows.